### PR TITLE
Fix broken links in cli docs

### PR DESCRIPTION
**Pull request by sukantoraymond**
Fix broken links in cli docs

______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/complex-transactions.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: How to build and send a complex Flow transaction from the command l
 **Simple Transactions**
 
 Sending a transaction using the Flow CLI can simply be 
-achieved by using the [send command documented here](/flow-cli/send-transactions/).
+achieved by using the [send command documented here](https://developers.flow.com/tools/flow-cli/send-transactions).
```
Comment (sideninja) [inline docs/complex-transactions.md:10]: shouldn't this be just `/tools/flow-cli...` without the domain?

Comment (sukantoraymond) [reply docs/complex-transactions.md:10]: we can use /tools/flow-cli.. but I just add the whole link just to be sure

Comment (sideninja) [reply docs/complex-transactions.md:10]: we should use only / part

Comment (sukantoraymond) [reply docs/complex-transactions.md]: addressed

**Review by sideninja — APPROVED** [review]
Thank you!

**Pull request merged**